<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PoTrupama</class>
 <widget class="QDialog" name="PoTrupama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Po trupama</string>
  </property>
  <widget class="QLabel" name="labelTrupa">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Trupa</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTrupa">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewGlumci">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>360</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableViewKomadi">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>10</y>
     <width>360</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPrikazi">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Prikazi</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonIzadji">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Izadji</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
